product_variant_inactive/README.rst

```diff
@@ -0,0 +1,45 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+ :alt: License: AGPL-3
+
+Module name
+===========
+
+This module was written to extend the functionality of Product Variants
+and allow you to make activate or deactivate products using the two buttons
+
+.. image:: https://raw.githubusercontent.com/akretion/product-attribute/8.0-product_variant_inactive/product_variant_inactive/img/img.png
+
+
+Installation
+============
+
+To install this module, you need to:
+
+*To install Product module
+
+For further information, please visit:
+
+* https://www.odoo.com/forum/help-1
+
+
```

Comment:
No block 'Installation' required => it seems the information here have no added value